webodf/lib/core/Zip.js

```diff
@@ -168,7 +168,7 @@ core.Zip = function Zip(url, entriesReadCallback) {
      */
     function createByteArray(successCallback, errorCallback) {
         try {
-            successCallback(/**@type{!Uint8Array}*/(zip.generate({type: "uint8array", compression: "DEFLATE"})));
+            successCallback(/**@type{!Uint8Array}*/(zip.generate({type: "uint8array", compression: "STORE"})));
```

Comment:
I've added tests now that show the issue appears when saving from an exisiting odf file. In new odt files, the mimetype is not compressed.